README.md

```diff
@@ -30,32 +45,56 @@ height="75px">
 
 ## Screenshots
 
-![](./packages/web/images/screenshots/screenshot-230602-a.png)
-
+![](./packages/web/images/screenshots/screenshot-2023-10-19.png)
 
 ## Project Setup
 
+### Stack
+- [NodeJS](https://nodejs.org/) (v18)
+- Back: [Express](http://expressjs.com/) (v4)
+- Front: [VueJS](https://vuejs.org/) (v3)
+
+### Description
 This repository is a monorepo, based on [npm 
 workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces/) composed of 
 two workspaces:
 
 - the ["web" workspace](./packages/web/README.md) contains the front-end implementations.
 - the ["backend" workspace](./packages/backend/README.md) contains the back-end implementations.
```

Comment:
todo: I just saw this this morning, the `data` workspace information is missing =>

```md
- the ["data" workspace](./packages/data/README.md) contains the datasets and common files for datasets builds.
```